src/main/java/taxi/controller/authentication/RegisterController.java

```diff
@@ -0,0 +1,57 @@
+package taxi.controller.authentication;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import taxi.lib.Injector;
+import taxi.model.Driver;
+import taxi.service.AuthenticationService;
+import taxi.service.DriverService;
+
+public class RegisterController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("taxi");
+    private final DriverService driverService = (DriverService) injector
+            .getInstance(DriverService.class);
+    private final AuthenticationService authenticationService =
+            (AuthenticationService) injector.getInstance(AuthenticationService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/authentication/register.jsp")
+                .forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String password = req.getParameter("password");
+        String confirmPassword = req.getParameter("confirmPassword");
+        if (!authenticationService.passwordValidation("password",
+                "confirmPassword")) {
```

Comment:
```suggestion
        if (!authenticationService.validatePassword(password,
                confirmPassword)) {
```
The name of the method must be a verb